packages/art-cli-tool/src/scaffold/ArtScaffold.ts

```diff
@@ -10,11 +10,23 @@ import { printInstructions } from './printLog';
 import { Scaffolds } from '../enums/Scaffolds';
 import { Stage } from '../enums/Stage';
 import executeNodeScript from 'art-dev-utils/lib/executeNodeScript';
+import { isArray, isObject } from 'art-lib-utils/dist/utils/lang';
```

Comment:
art-cli-tool does not dependence on art-lib-utils right now, add it as dependency.